.github/ISSUE_TEMPLATE/rfc.yml

```diff
@@ -1,138 +0,0 @@
-# Copyright (c) The OpenTofu Authors
-# SPDX-License-Identifier: MPL-2.0
-# Copyright (c) 2023 HashiCorp, Inc.
-# SPDX-License-Identifier: MPL-2.0
-
-# This RFC has been inspired by the Rust RFC template https://github.com/rust-lang/rfcs/blob/master/0000-template.md
-
-name: Submit RFC
-description: Submit a highly-structured change request for public discussion.
-labels: ["rfc", "pending-decision"]
-body:
-  - type: markdown
-    attributes:
-      value: |
-        Thank you for opening an RFC!
-        
-        RFCs are a highly-structured format for submitting change requests. All major changes to OpenTofu should first go through an RFC, to have a place for the community to discuss.
-        
-        The template has a lot of fields and is inspired by the Rust RFC template. Please take your time to fill it out carefully.
-        
-        Working experimental PoCs in a draft Pull Request are a very welcome attachment to an RFC, especially ones that are easy to check out and take for a spin.
```

Comment:
Do we want to still allow for WIP experimental PoCs? Those would probably have to be in a separate PR, than the PR created for the RFC itself

Reply:
[Include pseudocode or link to a Proof of Concept if applicable.](https://github.com/opentofu/opentofu/pull/1669/files#diff-04781075da4da83848eee431eddeb0c23ac0d71f861a4e35ae427cdf57de8929R25)

I could reword that as "Including pseudocode or link to a Proof of Concept is very welcome and recommended"

What do you think?

Reply:
I think that's fine, I just missed it